setup.py

```diff
@@ -8,6 +8,10 @@
 with codecs.open(os.path.join(here, 'README.md'), encoding='utf-8') as f:
     long_description = f.read()
 
+# Get requirements from requirements.txt file
+reqs = [line.strip() for line in open('requirements.txt').readlines()]
+requirements = list(filter(None, reqs))
+
 setup(
     # Application name:
     name="nidmviewer",
```

Comment:
I just remember that reading directly from files fails somewhere... so I've not been doing it.@chrisfilo what do you suggest?